test/unit_test/pass_test/test_remove_redundant_reshape.py

```diff
@@ -178,3 +179,30 @@ def test_pass(self):
 
         self.run_value_test(RemoveRedundantReshapePattern4())
         self.assertEqual(num_of_ops(self.exported_program(), ops.aten.reshape), 1)
+
+
+class RedundantReshapePattern5(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x):
+        reshape_shape = [4, 6, 8]
+        # AxBxC -> AxBxC
+        reshape = torch.reshape(x, reshape_shape)
+        return reshape
+
+    def get_example_inputs(self):
+        return (torch.randn(4, 6, 8),)
+
+
+class RemoveRedundantReshapePattern5Test(SinglePassValueTest):
+    def test_pass(self):
+        self.setup(RedundantReshapePattern5())
+
+        if Version(torch.__version__) <= Version("2.6.0.dev20241015"):
+            self.run_value_test(ConvertViewToReshape())
```

Comment:
Maybe #22 should be merged first?

Reply:
it's just out of curiousity.
why do you apply ConvertViewToReshape pass before RemoveRedundantReshapePattern5()?

Reply:
It's because torch 2.5 generates aten.view for `torch.reshape` function. So, to make aten.reshape, the pass has to be called.